README.md

```diff
@@ -1,6 +1,28 @@
 # Portfolio
-I am a part owner of a carpet cleaning company. I wanted to be able to analyze our biometrics to get a feel for how 
-my technicians were doing with additional services. As well as which cities or zipcodes seemed to make more money.
-This Portfolio comtains tables I have created as well as stored procedures created with SQL in order to Extract,
-Transform and Load the data I entered into a spreadsheet. This can be used for most any carpet cleaning company
-as well as altered for use in other services industries.
+<img src="https://user-images.githubusercontent.com/108885804/210106590-9f1c0477-dc49-4700-a4b3-0ef3154a7e87.jpg" height="200px"/>
+
+My name is Rhet Pulley. I am a part owner of a carpet cleaning company. I wanted to be able to analyze our biometrics to get a feel for how 
+my technicians were doing with the additional services we provide.
+
+<img src="https://user-images.githubusercontent.com/108885804/210104135-15d20986-a294-40d8-81ff-6223f1b12747.png" height="400px" />
+
+As well as which cities or zipcodes seemed to make more money.
+
+<img src="https://user-images.githubusercontent.com/108885804/210105899-3cb868a2-864d-4f1a-8168-e515272eb0e2.png" height="400px" />
```

Comment:
Make these smaller, smaller enough that it doesn't make it too blurry, but they don't need to see all the specifics